Backup.md

```diff
@@ -0,0 +1,29 @@
+## Solr Backup and Restore
+
+When using this backup/restore mechanism, Solr's replication handler is using Caringo Swarm as the storage layer for the backups. The implementation of the new backend repository is [here](https://github.com/xenit-eu/solr-backup).
+
+Taking a backup is trigerred by a [custom image](https://bitbucket.org/xenit/docker-solrbackup/src/master/), via a [central scheduler](https://github.com/crazy-max/swarm-cronjob). Cron expression is specified via container labels. Depending on an environment variable, backup can be taken only on first instance of solr service. 
```

Comment:
This looks like our internal implementation, which we might not want to put in the readme for docker-solr?

Reply:
Makes sense, I removed that section.